docs/src/P3Scheme.md

```diff
@@ -133,18 +133,29 @@ As a result ``q\_{ice}`` can be expressed as a sum of inclomplete gamma function
 where ``\Gamma \,(a, z) = \int_{z}^{\infty} \! t^{a - 1} e^{-t} \mathrm{d}D``
   and ``\Gamma \,(a) = \Gamma \,(a, 0)`` for simplicity.
 
+Within our solver, we approximate ``\mu`` from q/N and keep it constant throughout the solving step. We approximate ``\mu`` by an exponential function given by the q/N points corresponding to ``\mu = 6`` and ``\mu = 0``. This is shown below as well as how this affects the solvers ``\lambda`` solutions.
+
+```@example
+include("plots/P3LambdaErrorPlots.jl")
+```
+![](MuApprox.svg)
+
 An initial guess for the non-linear solver is found by approximating the gamma functions as a simple power function. 
 
 ```@example
 include("plots/P3ShapeSolverPlots.jl")
 ```
 ![](SolverInitialGuess.svg)
 
-This equation is given by ``(log(q_{approx}) - log(q1)) = slope (log(\lambda) - log(p1))``. Solving for ``q_{approx}`` we get `` q_{approx} = q_1 \frac{\lambda}{p_1} ^{slope}`` where `` slope = \frac{log(q1) - log(q2)}{log(p1) - log(p2)}``, ``p1`` and ``p2`` are defining ``\lambda`` values of the estimated line (we use p1 = 1e2, p2 = 1e6), ``q1 = q(p1)`` and ``q2 = q(p2)`` are the corresponding calculated q values for the given ``F_r`` and ``\rho_r`` values. 
+This equation is given by ``(log(q_{approx}) - log(q1)) = slope (log(\lambda) - log(p1))``. Solving for ``q_{approx}`` we get `` q_{approx} = q_1 \frac{\lambda}{p_1} ^{slope}`` where `` slope = \frac{log(q1) - log(q2)}{log(p1) - log(p2)}``, ``p1`` and ``p2`` are defining ``\lambda`` values of the estimated line decided off of the ``q/N`` value (described below), ``q1 = q(p1)/N`` and ``q2 = q(p2)/N`` are the corresponding calculated q values for the given ``F_r`` and ``\rho_r`` values. 
```

Comment:
I'm wondering if it would make it simpler to explain the initial guess by plotting log of lambda as a function of log of q/N. Instead of q/N as a function of lambda on a log scale. Also, for simpler notation we can use y=log(lambda) x=log(q/N) etc. Then I think it would be almost immediate to introduce the equation you use for the initial guess?

Reply:
This would also simplify the `p1`, `p2`, `q1`, `q2` notation. I think sometimes those `q` are actually `q/N`?